packages/flame_3d/lib/src/graphics/graphics_device.dart

```diff
@@ -177,7 +177,15 @@ class GraphicsDevice {
       );
 
       _renderTarget = gpu.RenderTarget.singleColor(
-        gpu.ColorAttachment(texture: colorTexture!, clearValue: clearValue),
+        gpu.ColorAttachment(
```

Comment:
assuming the vector expected was in the order RGBA

Reply:
Yes it is RGBA (probably)